src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -164,21 +156,15 @@ class RelayToTIRVisitor : public MixedModeMutator {
                                         ToArg(dilation_w),   ToArg(dilation_h),    ToArg(clip_min),
                                         ToArg(clip_max)};
 
-    // cmsis_nn_dims *input_dims (NHWC)
+    // layout NHWC
     Array<PrimExpr> input_shape = conv2d_call->args[0]->type_as<TensorTypeNode>()->shape;
-    Array<PrimExpr> input_dims = CMSISNNDimensions(input_shape);
 
-    // cmsis_nn_dims *filter_dims (OHWI for Conv2D and IHWO for depthwise)
+    // OHWI for Conv2D and IHWO for depthwise
```

Comment:
Same here, where is the information OHWI is used ?

Reply:
Same as above. Maybe I should explicitly say that this is for CMSIS-NN API?

Reply:
I think the information is that the following line produces a tensor is that layout and subsequent access assume that ?

Reply:
Yes, comment got displaced in the process of review and new edits :laughing: